cmd/deploy/deploy.go

```diff
@@ -278,7 +287,7 @@ func Deploy(ctx context.Context, at AnalyticsTrackerInterface, insightsTracker b
 	cmd.Flags().StringVarP(&options.Namespace, "namespace", "n", "", "overwrites the namespace where the development environment is deployed")
 	cmd.Flags().StringVarP(&options.K8sContext, "context", "c", "", "context where the development environment is deployed")
 	cmd.Flags().StringArrayVarP(&options.Variables, "var", "v", []string{}, "set a variable (can be set more than once)")
-	cmd.Flags().BoolVarP(&options.Build, "build", "", false, "force build of images when deploying the development environment")
+	cmd.Flags().BoolVarP(&options.Build, "build", "", true, "enable/disable building images. Default is true")
```

Comment:
```suggestion
	cmd.Flags().BoolVarP(&options.Build, "build", "", true, "enable/disable building images.")
```

cobra is already adding the default value